src/lib/HttpRequest.ts

```diff
@@ -1,101 +1,101 @@
-import net from 'net'
-import http from 'http'
-import { Address } from './peers/PeerInfo'
+// import net from 'net'
```

Comment:
Why is this all commented out? Are we no longer using HTTP?

Reply:
Oh wow that was quick! Actually the HttpRequest is no longer used so I've removed it. See latest file :+1:

Reply:
It is replaced by `GitClient`, new name since we're no longer using HTTP protocol